.project

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Unintended `git add`?

Reply:
Oops. Fixed. Added it to gitignore as well.